packages/@lwc/engine/src/__tests__/events.spec.ts

```diff
@@ -6,6 +6,7 @@
  */
 import { createElement, LightningElement } from '../';
 import { compileTemplate } from 'test-utils';
+import { registerDecorators } from '../framework/main';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.